src/renderers/__tests__/ReactUpdates-test.js

```diff
@@ -1174,6 +1174,29 @@ describe('ReactUpdates', () => {
     },
   );
 
+  it('uses correct base state for setState inside render phase', () => {
+    spyOn(console, 'error');
```

Comment:
Would be nice to check for specific message/count. Just in case.

Reply:
Updated